docs/server/upgrading.rst

```diff
@@ -3,170 +3,222 @@
 
-  Then continue with the upgrade process.
+.. _upgrading#setup-users:
 
-* Upgrade the Pootle package:
+Set up users
+------------
 
-  .. code-block:: bash
+Any accounts that do not have an email address registered will not be able to
+log in. You can set the email for a user using the :djadmin:`update_user_email`
+command.
 
-     (env) $ pip install --upgrade Pootle
+For example to set the email for user "admin" to "admin@example.com": 
 
-* Update your custom Pootle settings to adjust for any changes and to include
-  any new settings. Delete any obsolete settings. Check the :ref:`available
-  settings <settings#available>` as needed.
+.. code-block:: bash
 
-  .. note:: Running :djadmin:`pootle check` will highlight settings that have
-     been made obsolete or renamed.
+   (env) $ pootle update_user_email admin admin@example.com
 
-  .. note:: If you are upgrading from a version of Pootle that uses
-     :file:`localsettings.py` then you must :ref:`move your custom settings
-     <settings#customizing>` to a new location in order to ensure that Pootle
-     uses them.
 
-* Perform the database schema and data upgrade by running:
+As of Pootle 2.7 users must now verify their email in order to log in.
 
-  .. code-block:: bash
+You can automatically verify a user bypassing email verification using the
+:djadmin:`verify_user` command.
 
-     (env) $ pootle migrate
+For example to automatically verify the admin user:
 
+.. code-block:: bash
 
-* Reapply your custom changes to Pootle code, templates or styling. Read about
-  :doc:`customization of style sheets and templates
-  </developers/customization>` to adjust your customizations to the correct
-  locations and approach in the new release.
+   (env) $ pootle verify_user admin
 
-  .. note:: If you have customized the CSS styling or the JavaScript code you
-     will have to run the following commands to update the static assets:
+If you wish to automatically verify all of your existing users:
 
-     .. code-block:: bash
+.. code-block:: bash
 
-       (env) $ cd $pootle_dir/pootle/static/js/
-       (env) $ npm install
-       (env) $ npm update
-       (env) $ pootle webpack
-       (env) $ pootle collectstatic --noinput --clear -i node_modules -i *.jsx
-       (env) $ pootle assets build
+   (env) $ pootle verify_user --all
 
 
-     ``$pootle_dir`` is the directory where :command:`pip` installed Pootle. Its
-     location depends on your settings for :command:`pip`, but by default it
-     should be :file:`~/.virtualenvs/env/lib/python2.7/site-packages/`.
+.. _upgrading#start-rq:
 
+Start an RQ Worker
+------------------
 
-* Finally, restart your server.
+Before starting your upgraded Pootle application you will need an RQ worker
+running to process asynchronous tasks.
 
+You can start the worker with the following command:
 
-.. _upgrading#post-upgrade:
+.. code-block:: bash
 
-Post-upgrade adjustments
-------------------------
+   (env) $ pootle rqworker
```

Comment:
hmm - yep - ive been thinking about this - i might farm this out to a separate rq page.

It would also be useful to mention where the logs can be found